src/beachbot/manipulators/roarmm1.py

```diff
@@ -109,7 +134,7 @@ def set_joint_targets(self, qs):
                 "P2": qs[1],
                 "P3": qs[2],
                 "P4": qs[3],
-                "P5": qs[4],
+                "P5": percent,
```

Comment:
Did you figure out how the memory leak was occurring or just patch it like this? If just a patch, then we will potentially run into this with the other q[0:3] at some point.